app/components/Play/Play.js

```diff
@@ -1,32 +1,37 @@
 import React, { Component } from 'react';
-import { PickEmotion } from '../PickEmotion/PickEmotion';
+import PickEmotion from '../PickEmotion/PickEmotion';
 import ImageCapture from '../ImageCapture/ImageCapture';
 import { EmotionResults } from '../EmotionResults/EmotionResults';
+import { analyzeResponse } from '../../helper.js';
 
 export default class Play extends Component {
   constructor() {
     super()
     this.state = {
-      emotions: {}
+      emotions: []
     }
   }
 
-  analyzeEmotions(faceURL) {
+  analyzeEmotions(faceBlob) {
+    const body = faceBlob;
+
+    const headers = new Headers({
+      'Content-Type': 'application/octet-stream',
+      'Ocp-Apim-Subscription-Key': '5044b8a6914442cfbb74003672a494a0'
+    })
+
+
     fetch('https://westus.api.cognitive.microsoft.com/emotion/v1.0/recognize', {
         method: 'POST',
-        header: {
-            'Content-Type': 'application/json',
-            'Ocp-Apim-Subscription-Key': '3119bdf7116e4afebc7b8d910aa343d9'
-        },
-        body: `{'url': '${faceURL}'}`
+        headers: headers,
+        body: faceBlob
     })
     .then((response) => {
-      console.log('success');
       response.json();
```

Comment:
you need a `return` here.  you're converting the response to json and then never passing it along to the next part of the chain.  alternatively you could rewrite this in the shorthand `.then(response => response.json())`